home/models.py

```diff
@@ -136,6 +209,16 @@ class Meta:
         label = 'RSS Import'
 
 
+class PageCollectionBlock(core_blocks.StructBlock):
+    page_collection = snippet_blocks.SnippetChooserBlock(
+        required=False, target_model=PageCollection)
+
+    class Meta:
+        template = 'home/blocks/page_collection_block.html'
+        icon = 'snippet'
+        label = 'Page Collection'
+
+
```

Comment:
You could add a list block with PageCollectionBlock to be displayed on the section page